scripts/jwt_issuer/up.sh

```diff
@@ -14,7 +14,8 @@ jwt-issuer-up() {
 
     sudo chown $USER:$USER -R $JWT_ISSUER_WORKSPACE
 
-    export JWT_ISSUER="http://localhost:3000/token"
+    export JWT_TOKEN_ISSUER_URL="http://localhost:3000/token"
```

Comment:
Can you explain the difference between these two things? They're both URLs and I presume they both issue tokens so maybe we can think of a better naming scheme?

Reply:
Also not super keen on JWT_TOKEN because the T in JWT stands for token.

So we have JSON Web Token Token Issuer

Reply:
JWT_ISSUER is the name used in CCF for registering the JWT issuer.
JWT_TOKEN_ISSUER_URL is the actual url of the endpoint.

Reply:
it is very common to talk about JWT tokens. Just do a search for "JWT token".

Reply:
> JWT_ISSUER is the name used in CCF for registering the JWT issuer. JWT_TOKEN_ISSUER_URL is the actual url of the endpoint.

Ah thanks for the explanation, that makes sense

> it is very common to talk about JWT tokens. Just do a search for "JWT token".

Being common doesn't make it correct or less redundant, and I wouldn't claim it's more common that just JWT, [jwt.io](https://jwt.io) always just says JWT.

How about we use 
- `JWT_ISSUER_URL` which is the URL which issues JWTs
- `JWT_ISSUER_NAME` which is the name of the thing which issues JWTs (for CCF)
  - We could make it clearly a name by making it not look like a URL?

Reply:
I believe CCF requires this to be a url. Need to check this.

Reply:
If it needs to be a URL, could it just be the actual URL so we only need one variable instead of two?

Reply:
indeed.
{
  bool validate_issuer(
    const std::string& iss,
    const std::optional<std::string>& tid,
    std::string constraint)
  {
    LOG_DEBUG_FMT(
      "Verify token.iss {} and token.tid {} against published key issuer {}",
      iss,
      tid,
      constraint);

    const auto issuer_url = ::http::parse_url_full(constraint);

Reply:
> If it needs to be a URL, could it just be the actual URL so we only need one variable instead of two?

See https://github.com/microsoft/azure-privacy-sandbox-kms/pull/260#discussion_r1883672015